### DISCUSSION TITLE
Registering custom dataset that is in the COCO format

### DISCUSSION THREAD

**Discussion by PrabhjotKaurGosal** [facebookresearch/detectron2#2737] [Q&A] [answered]
When registering custom dataset that is already in the COCO format, is it not necessary to register the metadata separately -using getMetadats()? I m following the COLAB tutorial and it suggests the following.  . ..  Then, when I try to visualize my annotated data, it seems that I need the metadata as well. Any suggestions of how to get the metadata when registering using register_coco_instances(). 

if your dataset is in COCO format, this cell can be replaced by the following three lines:
from detectron2.data.datasets import register_coco_instances
register_coco_instances("my_dataset_train", {}, "json_annotation_train.json", "path/to/image/dir")
register_coco_instances("my_dataset_val", {}, "json_annotation_val.json", "path/to/image/dir")

Thanks!

**Comment by kadattack** [comment, marked as answer]
try MetadataCatalog.get("my_dataset_val")

**Reply by PrabhjotKaurGosal** [reply]
Thanks, it worked!
